### PR TITLE
Add CellProfiler analysis

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Add CellProfiler analysis

In this PR, I run CellProfiler 3D feature extraction using the EM data using the original format of the data. The SQLite output is uploaded to the repo.

### CODE REVIEW

File: image_profiling_env.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- conda-forge::typing-extensions
+# these are strict because that is how it is on the CellProfiler wiki (Jinja updated for nbconvert)
+- conda-forge::Jinja2=3.0.3
+- conda-forge::inflect=5.3.0
+- conda-forge::wxpython=4.1.0
+- conda-forge::sentry-sdk=0.18.0
+- pip:
+    - cellprofiler==4.2.5
+    - CytoTable==0.0.6
+    - pycytominer
```
Comment (MikeLippincott) [inline image_profiling_env.yml]: consider adding a version to pycytominer

File: utils/__pycache__/cp_parallel.cpython-38.pyc
Comment (MikeLippincott) [inline utils/__pycache__/cp_parallel.cpython-38.pyc:1]: consider adding `*.pyc` to your `.gitignore` file

File: utils/errors/__pycache__/exceptions.cpython-38.pyc
Comment (MikeLippincott) [inline utils/errors/__pycache__/exceptions.cpython-38.pyc:1]: Same here

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # ignore raw images
-raw_images/
+data/
+
```
Comment (MikeLippincott) [inline .gitignore:3]:
```suggestion

*.pyc

```

**Review by MikeLippincott — APPROVED** [review]
LGTM! Features extraction time!

File: image_profiling_env.yml (hunk 1 above)
Comment (jenna-tomkinson) [reply image_profiling_env.yml]: Added!

File: .gitignore (hunk 2 above)
Comment (jenna-tomkinson) [reply .gitignore:3]: Added!

File: utils/__pycache__/cp_parallel.cpython-38.pyc
Comment (jenna-tomkinson) [reply utils/__pycache__/cp_parallel.cpython-38.pyc:1]: Added to file!

File: utils/errors/__pycache__/exceptions.cpython-38.pyc
Comment (jenna-tomkinson) [reply utils/errors/__pycache__/exceptions.cpython-38.pyc:1]: Added!

**Comment by jenna-tomkinson** [comment]
@MikeLippincott Thanks for the review! I will be merging this now and we can move forward to preprocessing features! 🎉

**Pull request merged**